WordPress/Classes/Models/ReaderPost.m

```diff
@@ -178,5 +179,56 @@ - (NSURL *)featuredImageURLForDisplay
     return [self featuredImageURL];
 }
 
+- (SourceAttributionStyle)sourceAttributionStyle
+{
+    if (!self.sourceAttribution) {
+        return SourceAttributionStyleNone;
```

Comment:
Sorry i didn't realize this before. First check can be ditched, since if `sourceAttribution` is nil, or something else rather than `Post` or `Site`, it'll fall through to the last check, and return `SourceAttributionStyleNone`